css/index.css

```diff
@@ -310,12 +317,22 @@ footer {
     margin: 20px 0px;
   }
 
+  .docs-heading {
+    font-size: 1.8rem;
+  }
+
   .cards {
-    width: 85%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
![image](https://user-images.githubusercontent.com/66248039/151823693-ec01f55b-2ef0-42c2-a3a2-b068c6483670.png)
The content the last card/button looks a bit crowded. Can you fix this please? Either the card size can be increased or font-size can be reduced. Please check the breakpoints and make the changes accordingly!

Reply:
Edited check it out now!
Thank You 😊